tests/JavascriptTests.cmake

```diff
@@ -212,6 +220,7 @@ function(add_web_client_test case specFile)
   string(REPLACE ";" " " plugins "${plugins}")
 
   set_property(TEST ${testname} PROPERTY ENVIRONMENT
+    "PYTHONPATH=$ENV{PYTHONPATH}"
```

Comment:
We are freezing the pythonpath to whatever it is during config time? What's the reasoning for doing that?